common/item_fieldlist.h

```diff
@@ -203,3 +203,4 @@ F(procname)
 F(wornname)
 F(focusname)
 F(scrollname)
+F(subtype)
```

Comment:
should we order this better? Does it matter?

Reply:
I defer to you.  I'll add the accessor and change my PR tomorrow.